backend/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10
+FROM python:3
```

Comment:
I would prefer to leave this with the pinned 3.10 we had as there can be changes in Python versions.